src/components/Dropdown/Dropdown.tsx

```diff
@@ -141,12 +133,14 @@ export const Dropdown: FC<DropdownProps> = ({
 
     const showClear = !!activeItem && !!onClear;
 
-    const getDropdownBottomPosition = (dropDownSize: string) => {
-        return dropDownSize === DropdownSize.Small ? "tw-mb-2 tw-bottom-[34px]" : "tw-mb-2 tw-bottom-[60px]";
-    };
+    const leftPosition =
+        triggerRef.current && overlayRef.current
+            ? triggerRef.current?.getBoundingClientRect().right - overlayRef.current?.getBoundingClientRect().width
+            : "auto";
+    const topPosition = triggerRef.current ? triggerRef.current?.getBoundingClientRect().top - 3 : "auto";
```

Comment:
This does not work if we have Large dropdown triggers or if the dropdown is in a scrollable container. I think there's no way around using popper for that job.

Reply:
Updated!